experiments/conductor/cmd/runner/runner.go

```diff
@@ -156,10 +160,12 @@ func RunRunner(ctx context.Context, opts *RunnerOptions) error {
 	}
 
 	switch opts.command {
+	case -3:
```

Comment:
Is the plan to get each person generating a group of mocks to run this on their work set?

Reply:
Yes delegate updating their copy of branch-user.yaml 
The tool generates branch-new.yaml which can be inspected and renamed as the branch-user.yaml